recipes/lmdb/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.9.18" %}
+
+package:
+  name: lmdb
+  version: {{ version }}
+
+source:
+  fn: LMDB_{{ version }}.tar.gz
+  url: https://github.com/LMDB/lmdb/archive/LMDB_{{ version }}.tar.gz
+  sha1: 375e31bd0a4d8426e5fc4bfdb00d532c206d4f2b
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+test:
+  commands:
+    - mdb_copy -V                                    # [unix]
+    - mdb_dump -V                                    # [unix]
+    - mdb_load -V                                    # [unix]
+    - mdb_stat -V                                    # [unix]
+    - test -f ${PREFIX}/lib/liblmdb.a                # [unix]
+    - test -f ${PREFIX}/lib/liblmdb.so               # [unix]
```

Comment:
They are still normal Mac libraries, but yeah it is a strange thing that one sometimes sees.